File: backend/src/main/java/com/socialmeet/backend/dto/UserPhotoDTO.java
package com.socialmeet.backend.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import java.time.LocalDateTime;

/**
 * 用户照片DTO
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserPhotoDTO {

    private Long id;
    private Long userId;
    private String photoUrl;
    private Boolean isAvatar;
    private LocalDateTime uploadTime;
}

File: app/src/main/java/com/example/myapplication/dto/UserPhotoDTO.kt
package com.example.myapplication.dto

import java.time.LocalDateTime

/**
 * 用户照片DTO
 */
data class UserPhotoDTO(
    val id: Long? = null,
    val userId: Long,
    val photoUrl: String,
    val isAvatar: Boolean = false,
    val uploadedAt: LocalDateTime? = null,
    val createdAt: LocalDateTime? = null
)

/**
 * 上传照片响应DTO
 */
data class UploadPhotoResponse(
    val photoId: Long,
    val photoUrl: String,
    val isAvatar: Boolean
)
